### PR TITLE
test: add test from PR #3

### PR DESCRIPTION
**Pull request by bdura**
This PR adds tests from PR #3.

There are a few tests that address similar parts of the library, although they are not exactly the same. I also modified tests that rely on each other (within `cli/test_remote.py`, formerly `cli_test_base.py`)

### CODE REVIEW

File: docs/assets/images/projects.png
Comment (rmitsch) [inline docs/assets/images/projects.png:1]: Might be a bit misleading that the visuals all have `spaCy` references. It's fine for now, but perhaps we should change that at some point.

File: requirements.txt [hunk 1]
```diff
@@ -20,3 +20,14 @@ types-setuptools>=57.0.0
...
+mike
+mkdocs-bibtex
+mkdocs-gen-files
+mkdocs-literate-nav
+mkdocs-material
+mkdocstrings
+mkdocstrings-python
+mkdocs-glightbox
+mkdocs-jupyter
+mkdocs-charts-plugin
```
Comment (rmitsch) [inline requirements.txt:33]: No pins?

**Review by rmitsch — COMMENTED** [review]
Why does this PR include the markdown files as well if the goal is adding tests?

**Comment by bdura** [comment]
> Why does this PR include the markdown files as well if the goal is adding tests?

Ah, I merged the documentation PR because one of the tests relied on the output of the help... I should have put a TODO instead, sorry about that.

File: docs/assets/images/projects.png
Comment (bdura) [reply docs/assets/images/projects.png:1]: That's true, we should keep that in mind.

File: requirements.txt (hunk 1 above)
Comment (bdura) [reply requirements.txt]: Thanks.

**Pull request closed without merging**